### PR TITLE
Add scroll tracking to foreign travel advice pages

### PR DESCRIPTION
**Pull request by andysellick**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What
Adds scroll tracking to the pages listed on https://www.gov.uk/foreign-travel-advice (but not that page itself). Should add tracking for all pages for all countries, for example on:

- https://www.gov.uk/foreign-travel-advice/afghanistan
- https://www.gov.uk/foreign-travel-advice/afghanistan/coronavirus
- https://www.gov.uk/foreign-travel-advice/afghanistan/safety-and-security

(etc.)

Uses the new AutoScrollTracker module, initialised as a meta tag, configured to track headings. This is included in the latest version of the gem, which has already been added to government-frontend (this PR rebased to match).

## Why
We want to understand user behaviour on these pages in order to improve them.

## Visual changes
None.

Trello card: https://trello.com/c/JREgQ2iv/579-spike-into-tracking-enhancements

### CODE REVIEW

**Review by kevindew — APPROVED** [review]
Code looks good 👍 

I had some trouble testing this out because Static on production doesn't yet contain this file (so `./startup --live` didn't work). Thought I'd mention so you don't merge this and find it doesn't work in prod because Static is behind.

**Comment by andysellick** [comment]
> I had some trouble testing this out because Static on production doesn't yet contain this file (so `./startup --live` didn't work). Thought I'd mention so you don't merge this and find it doesn't work in prod because Static is behind.

Oh of course 🤦 . I'll make sure static has the new version so you can check it before I merge.

**Comment by kevindew** [comment]
> > I had some trouble testing this out because Static on production doesn't yet contain this file (so `./startup --live` didn't work). Thought I'd mention so you don't merge this and find it doesn't work in prod because Static is behind.
> 
> Oh of course 🤦 . I'll make sure static has the new version so you can check it before I merge.

Thanks, don't worry about me needing to check it though. I used integration yesterday to try it out.

**Pull request merged**
